docs/examples/rrlyr-period.ipynb

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
<p>I would recommend adding a markdown cell or two introducing the example and briefly describing the objective of the code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/235/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>